scripts/accounts/aws/verify-account-policies.sh

```diff
@@ -0,0 +1,115 @@
+    echo "  -a     [Optional] AWS administrator access key"
+    echo "  -s     [Optional] AWS administrator secret key"
+    echo "  -r     [Optional] AWS region"
+    exit
+fi
+
+#Argument check
+if [ -z "$username" ] || { [ -z "$policies" ] && [ -z "$policies_file" ]; }
+then
+    echo "Missing parameters, -u and -p or -f flags are mandatory."
+    echo "Use -h flag to display help."
+    exit
+fi
+
+#AWS credentials setup
+if [ -n "$access_key" ]  && [ -n "$secret_key" ];
+then
+    echo "Setting up your AWS credentials.."
+    export AWS_ACCESS_KEY_ID=$access_key
+    export AWS_SECRET_ACCESS_KEY=$secret_key
+fi
+
+#AWS region
+if [ -n "$region" ];
+then
+    echo "Setting up your AWS region.."
+    export AWS_DEFAULT_REGION=$region
+fi
+
+#AWS default output
+export AWS_DEFAULT_OUTPUT=json
+
+#Check if AWS credentials are valid
+aws sts get-caller-identity &> /dev/null
+if ! [ $? -eq 0 ]
+then
+    echo "Invalid AWS credentials. Please use -a -s flags to setup correctly.";
+    exit 
+fi
+
+#Get user groups
+echo "Getting user groups.."
+user_groups=($(aws iam list-groups-for-user --user-name $username --query 'Groups[].[GroupName]' --output text))
+
+#Loop user groups and get policies from them
+echo "Getting policies attached to user groups.."
+groups_policies=()
+for group in ${user_groups[@]}
+do
+    cleangroup=$(echo $group | tr -cd '\11\12\15\40-\176')
+    groups_policies+=($(aws iam list-attached-group-policies --group-name $cleangroup --query 'AttachedPolicies[].[PolicyArn]' --output text))
+done
+
+#Get user-specific policies
+echo "Getting user-specific policies.."
+user_policies=($(aws iam list-attached-user-policies --user-name $username --query 'AttachedPolicies[].[PolicyArn]' --output text))
+all_policies=( "${groups_policies[@]}" "${user_policies[@]}")
+
+#Inline policies check
+if [ -n "$policies" ];
+then
+    echo "Checking inline provided policies.."
+    IFS=',' read -ra policies_array <<< "$policies"
+    for policy_to_check in "${policies_array[@]}"; do
+        policy_to_check=$(echo $policy_to_check | tr -cd '\11\12\15\40-\176')
+        policy_exists=$(printf '%s\n' "${all_policies[@]}" | grep "$policy_to_check")
+        if [ -n "$policy_exists" ];
+        then
+            echo "OK        $policy_to_check"
+        else
+            echo "FAILED        $policy_to_check"
+        fi
+    done
+fi
+
+#File policies check
+if [ -n "$policies_file" ];
+then
+    echo "Checking file provided policies.."
+    IFS=$'\r\n' GLOBIGNORE='*' command eval  'policies_file_array=($(cat ${policies_file}))'
+    for policy_to_check in "${policies_file_array[@]}"
+    do
+        policy_to_check=$(echo $policy_to_check | tr -cd '\11\12\15\40-\176')
+        policy_exists=$(printf '%s\n' "${all_policies[@]}" | grep "$policy_to_check")
+        if [ -n "$policy_exists" ];
+        then
+            echo "OK        $policy_to_check"
+        else
+            echo "FAILED        $policy_to_check"
```

Comment:
```suggestion
            echo "OK        $policy_to_check"
        else
            echo "FAILED    $policy_to_check"
```

Try to use green and red colours for OK and FAILED words. Check other PR to see how it is done.